### PR TITLE
Uptycs rm argument

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/32208

### CODE REVIEW

File: Packs/Uptycs/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Uptycs
+- ***breaking changes***: removed the *filename* argument from the ***uptycs-post-threat-source***
```
Comment (ShahafBenYakir) [inline Packs/Uptycs/ReleaseNotes/1_0_3.md:4]:
```suggestion
Removed the *filename* argument from the ***uptycs-post-threat-source***.
```
I think this is not a breaking change

**Comment by bkschmoll-uptycs** [comment]
Hi all, why are we removing this argument?

**Comment by ShahafBenYakir** [comment]
@bkschmoll-uptycs The argument was not used in the code and is not necessary so it's just confusing.

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by moishce** [comment]
Need force merge because we removed an argument that was not used in the code from the YML file.

**Pull request merged**